modules/@apostrophecms/doc-type/index.js

```diff
@@ -375,6 +390,10 @@ module.exports = {
           arrangeFields: self.apos.schema.groupsToArray(self.fieldsGroups)
         });
         if (self.options.slugPrefix) {
+          if (self.options.slugPrefix === 'deduplicate-') {
+            const req = self.apos.task.getReq();
+            throw self.apos.error('invalid', req.__('The deduplicate- slug is reserved.'));
```

Comment:
Oof. Between you and Miro I think I'd better open a ticket to provide at least halfassed bc wrapper for `__`. I'll do that.

Reply:
(This is not a blocker or complaint.)

Reply:
I can leave the __ off if you'd prefer. There'll be a localization sweep later anyway.